TAF/testScenarios/functionalTest/V2-API/core-metadata/deviceprofile/PATCH-Negative-Basicinfo.robot

```diff
@@ -4,26 +4,30 @@ Resource     TAF/testCaseModules/keywords/core-metadata/coreMetadataAPI.robot
 Suite Setup  Run Keywords  Setup Suite
 ...                        AND  Run Keyword if  $SECURITY_SERVICE_NEEDED == 'true'  Get Token
 Suite Teardown  Run Teardown Keywords
-Force Tags  Skipped
+Force Tags  v2-api
 
 *** Variables ***
 ${SUITE}          Core Metadata Device Profile PATCH Basicinfo Negative Test Cases
 ${LOG_FILE_PATH}  ${WORK_DIR}/TAF/testArtifacts/logs/core-metadata-deviceprofile-patch-basicinfo-negative.log
 
 *** Test Cases ***
-ErrProfileBasicInfoPATCH001 - Update basicinfo with name validation error
-    #Empty profile name
-    Given Generate A Device Profile Sample for basicinfo
-    And Set To Dictionary  ${deviceProfile}[0][profile]  name=${EMPTY}
-    When Update Basicinfo ${deviceProfile}
+ErrProfileBasicInfoPATCH001 - Update basicinfo with Empty profile name
+    # Empty profile name
+    Given Generate a basicinfo sample for updating
+    And Set To Dictionary  ${basicinfoProfile}[0][basicinfo]  name=${EMPTY}
+    When Update basicinfo ${basicinfoProfile}
     Then Should Return Status Code "400"
-    [Teardown]  Delete Device Profile By Name
-
-ErrProfileBasicInfoPATCH002 - Update basicinfo with invalid profile name
-    #Non-existent profile name
-    Given Generate A Device Profile Sample for basicinfo  Test-Profile-5
-    And Set To Dictionary  ${deviceProfile}[0][profile]  name=Non-existent
-    When Update Basicinfo ${deviceProfile}
-    And Should Return Status Code "404"
-    [Teardown]  Delete Device Profile By Name
+    And Should Return Content-Type "application/json"
+    And Response Time Should Be Less Than "${default_response_time_threshold}"ms
+    [Teardown]  Delete device profile by name  Test-Profile-1
```

Comment:
You have set`Test-Profile-1` to `test_profile` test variable in the `Generate a basicinfo sample for updating`, so use `test_profile` here instead.
```suggestion
    [Teardown]  Delete device profile by name  ${test_profile}
```